src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocNonTightHtmlTags2.java

```diff
@@ -19,10 +19,10 @@
  */
 // violation 9 lines above 'tag BODY_TAG_START'
 // violation 9 lines above 'tag P_TAG_START'
-// 2 violations 8 lines above
+// 2 violations 8 lines above:
 //                            'Unclosed HTML tag found: p'
 //                            'tag P_TAG_START'
-// 2 violations 10 lines above
+// 2 violations 10 lines above:
 //                            'tag LI_TAG_START'
 //                            'tag P_TAG_START'
```

Comment:
What forces us to use crazy indentation?

Reply:
My brain :).
For some reason I found it convenient and easy to read to let text to vertically after `:`

Reply:
Yes, indentation helps for sure. This much isn’t a requirement though?

Reply:
indentation can be any, its completely skipped during matching by pattern.